packages/foreman/tfm/tfm.spec

```diff
@@ -19,12 +21,13 @@
 Summary: Package that installs %scl
 Name: %scl_name
 Version: 5.0
-Release: 8%{?dist}
+Release: 9%{?dist}
 License: GPLv2+
 Group: Applications/File
 Source0: README
 Source1: LICENSE
 Source2: tfm.attr
+Source3: nodejs-symlink-deps
```

Comment:
@ekohl I had to add this in order to fix this bug I found -- https://bugzilla.redhat.com/show_bug.cgi?id=1755675

Reply:
That increases my suspicion that we're the only people building RPMs on top of a NodeJS SCL

Reply:
Absolutely